.circleci/config.yml

```diff
@@ -96,23 +96,14 @@ package_steps: &package_steps
           set +e
           export NVM_DIR="/opt/circleci/.nvm"
           [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-          nvm install v12
-          nvm use v12
+          nvm install v14
+          nvm use v14
           npm install -g npm
 
     # Run the package tests on each currently supported node version. See:
     # https://github.com/badges/shields/blob/master/badge-maker/README.md#node-version-support
     # https://nodejs.org/en/about/releases/
 
-    - run:
-        environment:
-          mocha_reporter: mocha-junit-reporter
-          MOCHA_FILE: junit/badge-maker/v10/results.xml
-          NODE_VERSION: v10
-          CYPRESS_INSTALL_BINARY: 0
-        name: Run package tests on Node 10
-        command: scripts/run_package_tests.sh
-
```

Comment:
I am a bit confused about this. We dropped node 10 testing back in https://github.com/badges/shields/pull/6570/files

I'm guessing you must have taken this working branch off an outdated master but I'd expect that the PR status would be showing merge conflicts and it isn't.. Any chance you could have a look at what is going on here?

Reply:
Hmmm, I took from when I started work on #6651, and indeed  #6570 went through in the meantime.

Reply:
From #6570:

> We dropped compatibility for node 10 in version 3.0.0 of badge-maker but we've still been running the tests on it.

This is not the case. We dropped support for versions older than Node 10, but at the time of the 3.0.0, Node 10 was still supported. This PR therefore does take care of the actual dropping of Node 10 by updating _package.json_.

Reply:
Ah yes. You're totally right. We dropped node < 10 in 3.0.0.

Agree lets drop it here :+1:  but I think that means the next badge-maker release needs to be a major version bump since it drops a node version.